tasks/Module 4/task_7.py

```diff
@@ -0,0 +1,21 @@
+"""Excel-to-CSV Converter."""
+import os
+import csv
+import openpyxl
+
+
+def xl_to_csv(xl_file):
+    """Excel-to-CSV Converter."""
+    os.chdir(os.path.dirname(xl_file) if os.path.dirname(xl_file) else '.')
+    work_book = openpyxl.load_workbook(xl_file)
+    for sheet in work_book.worksheets:
+        output_file = f'{os.path.splitext(os.path.basename(xl_file))[0]}_{sheet.title}.csv'
+        with open(output_file, 'w', newline='') as csv_file:
```

Comment:
Why do you need `newline=''` here?

Reply:
Because of this: https://stackoverflow.com/questions/3348460/csv-file-written-with-python-has-blank-lines-between-each-row

Reply:
Good to know, maybe I should do coding on Windows from time to time ;)